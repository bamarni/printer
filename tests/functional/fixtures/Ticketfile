INIT

ALIGN RIGHT
FONT C

PRINT qwertz asdfgh

CUT

COLOR RED
CHARSET PC850

PRINTLF Être ou ne pas être
LF 4

CUT FULL
