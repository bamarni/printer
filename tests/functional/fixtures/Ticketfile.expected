@a2M2qwertz asdfghVA0r1tÒtre ou ne pas ˆtredVA0